package.json

```diff
@@ -40,13 +40,13 @@
     "exit": "~0.1.1",
     "findup-sync": "~0.3.0",
     "glob": "~7.0.0",
-    "grunt-cli": "1.0.0-rc1",
+    "grunt-cli": "1.0.1",
     "grunt-legacy-log": "1.0.0-rc1",
     "grunt-legacy-util": "1.0.0-rc1",
     "iconv-lite": "~0.4.13",
     "js-yaml": "~3.5.2",
     "minimatch": "~3.0.0",
-    "nopt": "~3.0.6",
+    "nopt": "~1.0.10",
```

Comment:
We need fixed in the new nopt. We are trying to make sense of the changes and make them compatible

cc @shama